css/style.css

```diff
@@ -68,12 +121,87 @@ body {
 	padding-right: 5px;
 }
 
+/* Image slideshow CSS begins */
```

Comment:
The image slideshow is something I added in this version because I had some time while waiting for stakeholder feedback. There is quite a bit of css for it